README.md

```diff
@@ -224,7 +224,7 @@ Once registered you can define the dockerfile to use with `-f Dockerfile.aarch64
 
 ## Versions
 
-* **12.02.20:** - Correct Name, Nzbhydra2 -> Nzbhydra2.
+* **14.04.20:** - Correct Name, Nzbhydra2 -> Nzbhydra2.
```

Comment:
shouldn't this be `Hydra2 to Nzbhydra2`?

Reply:
Yup. Silly mass replace messed up.